index.html

```diff
@@ -0,0 +1,128 @@
+<html>
+  <head>
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
```

Comment:
I think it would be better to use add meta charset tags in this section because the "meta charset" tag in HTML specifies the character encoding used in the document. This tag tells the web browser which character set to use to properly display the text content of the webpage. Please kindly add them to meet the requirements 👍